modules/tinymce/src/core/main/ts/fmt/CaretFormat.ts

```diff
@@ -165,6 +165,29 @@ const cleanFormatNode = (editor: Editor, caretContainer: Node, formatNode: Eleme
   }
 };
 
+const normalizeNbspsBetween = (editor: Editor, caretContainer: Node | null) => {
+  const handler = () => {
+    if (caretContainer && !editor.dom.isEmpty(caretContainer)) {
+      Traverse.prevSibling(SugarElement.fromDom(caretContainer)).each((node) => {
+        if (NodeType.isText(node.dom)) {
+          node.dom.data = node.dom.data.replace(Unicode.nbsp, ' ');
+        } else {
+          PredicateFind.descendant(node, (e) => NodeType.isText(e.dom)).each((textNode) => {
+            if (NodeType.isText(textNode.dom)) {
+              textNode.dom.data = textNode.dom.data.replace(Unicode.nbsp, ' ');
+            }
+          });
+        }
+      });
+    }
+  };
+  editor.once('input', (e) => {
+    !e.isComposing ? handler() : editor.once('compositionend', () => {
```

Comment:
Use an if statement I think we should only use ternaries for expressions that return something other than void.

Reply:
changed to a normal if, I did with a ternary operator just to make it less "ugly", but in the end is "ugly" anyway, so switch to if is a good idea :sweat_smile:

Reply:
I thought TypeScript could be configured to error on this. Maybe it was eslint.